README.md

```diff
@@ -5,34 +5,34 @@ A wrapper around SonarScanner CLI, available on PyPI.
 
```

Comment:
On line 1, should we use the `pysonar` name as well, since we say it's available on PyPI (and therefore we might want to use the name under which it will be known on PyPI)?

Reply:
Oh yes good catch! so
```
# pysonar
A wrapper around SonarScanner CLI, available on PyPI.
```